modules/calendar/calendar_creator.py

```diff
@@ -0,0 +1,41 @@
+from typing import Iterable
+from utils.sql_fetcher import SqlFetcher
+import psycopg2.extensions as sql
+from .calendar_service import CalendarService
+from .campus import Campus
+
+
+class CalendarCreator:
+	def __init__(
+		self, service: CalendarService, conn: sql.connection, sql_fetcher: SqlFetcher
+	):
+		self.__conn = conn
+		self.__sql_fetcher = sql_fetcher
+		self.__service = service
+
+	async def create_class_calendars(self, year: int):
+		"""Create a calendar for each campus and add it to the database"""
+		for campus in await self.__get_campuses():
+			# create calendar
+			calendar = self.__service.create_calendar(f"JCT CompSci {campus.name} {year}")
+			# update class in database
+			await self.__add_calendar(calendar.id, campus.id, year)
```

Comment:
```suggestion
			await self.__add_calendar(calendar.id, campus.id, year)
```
This function doesn't need to be a coroutine